File: app/src/main/java/com/wealoha/social/beans/FeedGetData.java
package com.wealoha.social.beans;

import java.util.List;
import java.util.Map;

import com.wealoha.social.api.common.dto.ImageDTO;
import com.wealoha.social.api.common.dto.VideoDTO;

public class FeedGetData extends ResultData {

	public List<PostDTO> list;

	public Map<String, ImageDTO> imageMap;
	public Map<String, VideoDTO> videoMap;

	public Map<String, Integer> commentCountMap;

	public Map<String, Integer> likeCountMap;

	public Map<String, UserDTO> userMap;

	public String nextCursorId;

}

File: app/src/main/java/com/wealoha/social/beans/FeedGetData.kt
package com.wealoha.social.beans

data class FeedGetData(
    var list: List<PostDTO>? = null,
    var imageMap: Map<String, ImageCommonDto>? = null,
    var videoMap: Map<String, VideoCommonDTO>? = null,
    var commentCountMap: Map<String, Int>? = null,
    var likeCountMap: Map<String, Int>? = null,
    var userMap: Map<String, UserDTO>? = null,
    var nextCursorId: String? = null
) : ResultData() {

}